docs/UserGuide.md

```diff
@@ -430,8 +430,9 @@ Example: `deleteloan 1`
 Provides visual analytics of a client's loan records based on three indices: Reliability, Impact, and Urgency.
 * The Reliability index is defined as the ratio of overdue loans to the total number of loans.
 * The Impact index is defined as the ratio of the average loan value to the maximum loan value.
-* The Urgency index is defined as the ratio of the number of days between the latest return date among all active loans and the latest return date among this particular client's active loans to the number of days between the latest return date among all active loans and the current date.
-  * If the latest return date among all active loans or the latest return date among this particular client's active loans is earlier than the current date, the Urgency index will be 1.
+* The Urgency index is defined as the ratio of the number of days between the earliest return date among this particular client's active loans and the current date to the number of days between the earliest return date among all active loans and the current date.
```

Comment:
Ooh, this might be a bit of a mouthful. Could we possibly change this in the future to something like `if E1 is the earliest return date... and E2 is the earliest return date, then the Urgency Index is defined as the ratio of E1 to E2`?